### PR TITLE
Add cloudmesh-openapi

### PR DESCRIPTION
**Pull request by laszewsk**
Adding cloudmesh-openapi

### CODE REVIEW

File: _data/tools.yml [hunk 1]
```diff
@@ -1,3 +1,14 @@
+- name: Cloudmesh OpenAPI
+  category: converters
+  language: Python
+  description:
+    Transform Python function specifications into OpenAPI services
+  link: https://github.com/cloudmesh/cloudmesh-openapi
+  v2: false
+  v3: true
+
+- name: Google Gnostic
```
Comment (matthewtrask) [inline _data/tools.yml:10]: is this supposed to be there?

**Comment by laszewsk** [comment]
This is an OpenAPI tool to create an OpenAPI service from a python function. The advantage is that users only need to know how to write a python function. This function is then translated into an OpenAPI service. 

The reason this is important is that no knowledge of OpenAPI is needed which lots of people I work with have not. For example, I work with a group affiliated with NIST that leverages this approach.

**Comment by laszewsk** [comment]
Oh, I see, I by mistake may have duplicated the -name: from the entrie I copied mine from ...

I fixed it

**Comment by laszewsk** [comment]
i will look into it soon

**Comment by daveshanley** [comment]
Is the project still active, looks very quiet over there?

**Comment by AndrewWalsh** [comment]
Following @daveshanley 's comment, project is rather quiet and hasn't received updates in some years. Closing.

**Pull request closed without merging**